### PR TITLE
LB-596, LB-593, LB-581: Update frontend to show different time ranges for artist stats

### PR DESCRIPTION
**Pull request by ishaanshah**
We now support multiple time ranges for top user artists, this PR aims to:
- Add support for viewing these statistics.
- Update tests

### CODE REVIEW

**Comment by ishaanshah** [comment]
Peek 2020-05-27 19-44
@paramsingh @mayhem @MonkeyDo Please have a look at the UI, I would love to hear some feedback

File: listenbrainz/webserver/static/js/src/stats/UserArtists.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```
Comment (paramsingh) [inline listenbrainz/webserver/static/js/src/stats/UserArtists.tsx:1]: why is this needed?
